src/env.c

```diff
@@ -303,3 +304,134 @@ char **env_update_home(char **envp, const char *dir) {
 
 	return envp;
 }
+
```

Comment:
We need to avoid this source code duplication. Please factor these two functions into separate source files cronie_common.c in root of the cronie source tree and compile and link it both into cronie and anacron.
Prototype declaration should be in cronie_common.h.